internal/gojs/misc_test.go

```diff
@@ -47,14 +50,27 @@ func Test_stdio(t *testing.T) {
 func Test_stdio_large(t *testing.T) {
 	t.Parallel()
 
+	// Large stdio will trigger GC which will trigger events.
```

Comment:
we had an existing heavier test (which flaked in go 1.17 until a recent workaround), so re-purposed it to also prove that the poll logging scope works.